kafka-connect-upload-source/src/main/java/org/radarbase/connect/upload/converter/altoida/AltoidaBlinkCsvProcessor.kt

```diff
@@ -20,9 +20,7 @@ import org.radarbase.connect.upload.converter.StatelessCsvLineProcessor
 import org.radarbase.connect.upload.converter.TopicData
 import org.radarcns.connector.upload.altoida.AltoidaBlink
 
-class AltoidaBlinkCsvProcessor : StatelessCsvLineProcessor() {
-    override val fileNameSuffix: String = "_BLINK.csv"
-
+class AltoidaBlinkCsvProcessor(override val fileNameSuffix: String = "_BLINK.csv") : StatelessCsvLineProcessor() {
```

Comment:
I've added the `fileNameSuffix` as a parameter so we can reuse the converters since mostly the main changes were the filenames.